testcafe/tests/pages/drive-viewer/drive-viewer-model.js

```diff
@@ -81,15 +82,11 @@ export default class Page {
         .expect(this.viewerNavNext.exists)
         .notOk('Next button on last picture')
     } else {
-      const fileurl = await getPageUrl()
       await t
         .hover(this.viewerNavNext) //not last photo, so next button should exists
         .expect(this.viewerNavNextBtn.visible)
         .ok('Next arrow does not show up')
         .click(this.viewerNavNextBtn)
-
-      const file2url = await getPageUrl()
-      await t.expect(fileurl).notEql(file2url)
```

Comment:
Because we don't need it anymore as we'll take screenshots after clicking on the 'next' button (cf. https://github.com/cozy/cozy-drive/blob/998d6fd6fddf76cc4b6e31b8f35add01bd77c5fe/testcafe/tests/pages/drive-viewer/drive-viewer-model.js#L122-L127

Reply:
👍